anvil/src/winit.rs

```diff
@@ -345,7 +332,10 @@ pub fn run_winit(log: Logger) {
         }
 
         // Send frame events so that client start drawing their next frame
-        state.space.send_frames(start_time.elapsed().as_millis() as u32);
+        state
+            .space
+            .elements()
+            .for_each(|window| window.send_frame(start_time.elapsed().as_millis() as u32));
```

Comment:
small nit, we could move that function so that we can share the logic between the backends

Reply:
Did that and then forgot to push it..
https://github.com/Smithay/smithay/pull/705/commits/df64e7ebdad6be62e755600114c435c1bf4cfad3